main.py

```diff
@@ -124,7 +116,7 @@ def main():
     for element in json_series:
         series_id = element['id']
         series_name = element['title']
-        process_season(series_id)
+        z += process_season(series_id, series_name)
```

Comment:
Globals actually don't seem as bad in python as in C++ (I forget why globals are even bad in the first place) but if I were to do this without globals I would do it something like this. Keep a local counter for each call of `process_season` and then add it to the counter total after `process_season` returns.